java/src/jmri/jmrix/loconet/LnTurnout.java

```diff
@@ -289,13 +289,17 @@ private void setKnownStateFromOutputStateOddReport() {
         newCommandedState(CLOSED + THROWN);
         if (getFeedbackMode() == MONITORING || getFeedbackMode() == DIRECT) {
             newKnownState(CLOSED + THROWN);
+        } else if (getFeedbackMode() == EXACT) {
+            newKnownState(INCONSISTENT);
```

Comment:
I think this should be CLOSED+THROWN as with MONITORING and DIRECT.  That's what the report says.  INCONSISTENT is for a separate case.